docs/fast_data/inputs_and_outputs.md

```diff
@@ -379,151 +487,131 @@ value: |
       }
     }
   }
+}
 ```
-
 </p>
 </details>
 
-**AsyncApi specification**:
-
-```yaml
-asyncapi: 2.4.0
-info:
-  title: Projection Update Producer
-  version: "1.0.0"
-channels:
-  projectionUpdateChannel:
-    subscribe:
-      message:
-        name: Projection Update
-        payload:
-          type: object
-          additionalProperties: false
-          properties:
-            key:
-              type: object
-            value:
-              type: object
-              additionalProperties: false
-              properties:
-                operationType:
-                  type: string
-                  enum: ["INSERT", "UPDATE", "DELETE", "UPSERT"]
-                operationTimestamp:
-                  type: number
-                documentId:
-                  type: string
-                projectionName:
-                  type: string
-                primaryKeys:
-                  type: Array
-                source:
-                  type: string
-                before:
-                  type: object
-                  additionalProperties: true
-                after:
-                  type: object
-                  additionalProperties: true
-                __internal__kafkaInfo:
-                  type: object
-                  additionalProperties: false
-                  properties:
-                    topic:
-                      type: string
-                    partition:
-                      type: integer
-                    offset:
-                      type: string
-                    key: {}
-                    timestamp:
-                      type: string
-          required: ["key", "value"]
-```
-
-#### Topic naming convention
+## Single View
 
-**producer** Real Time Updater
+This section covers the inputs and outputs concerning the Single View's aggregation.
 
-`<tenant>.<environment>.<mongo-database>.<collection>.pr-update`
+### Projection Changes
 
-An example:
+**Channel**: MongoDB
 
-```sh
-test-tenant.PROD.restaurants-db.reviews-collection.pr-update
-```
-
-### Projection Change
+**Producer**: Real Time Updater
 
-A `Projection Change` is an event that informs the listener that a Single View should be updated.
-It can be stored on either MongoDB or Kafka, depending on your architecture.
+**Consumer**: Single View Creator
 
-Its value has the following fields:
+**Description**: The Projection Changes or `pc` informs the listener (generally the Single View Creator) that a Single View should be updated.
```

Comment:
I would add to the description that this event is generated only when the env var (which I don't remember the name) is set to MONGO instead of KAFKA

Reply:
I believe you're confusing it with the env of the Single View Creator which tells the service from where to read the projection changes (mongo or kafka). The projection changes itself are always emmitted by the RTU unless it is explicitly disabled with the `PROJECTIONS_CHANGES_ENABLED` set to `false`

Reply:
yep you are right, I was confusing it with the SVC